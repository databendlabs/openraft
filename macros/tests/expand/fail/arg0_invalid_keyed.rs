fn main() {
    openraft_macros::expand!(
        !FOO,
        (K, T, V) => {K; T; V;},
    );

    openraft_macros::expand!(
        FOO,
        (K, T, V) => {K; T; V;},
    );
}
