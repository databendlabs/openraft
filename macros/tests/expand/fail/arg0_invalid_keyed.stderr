error: Expected KEYED
 --> tests/expand/fail/arg0_invalid_keyed.rs:3:10
  |
3 |         !FOO,
  |          ^^^

error: Expected KEYED
 --> tests/expand/fail/arg0_invalid_keyed.rs:8:9
  |
8 |         FOO,
  |         ^^^
